masterfabric-mobile/swift-camp
# CL - VStack Viper implementation

## 📋 PR Description

This PR introduces the implementation of a VStack view using the VIPER architecture pattern. The key components (View, Presenter, Wireframe) have been defined and structured to follow a modular approach for better separation of concerns and scalability. Below are the main highlights:

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Launch the application.
2. Navigate to the VStackView screen from HomeView by clicking the "VStackView" button.
3. Test the back button to ensure it navigates correctly to the HomeView.


---

## 🔗 Related Links

- Documentation: https://medium.com/@dickareynaldisiahaan/viper-architecture-in-swift-building-robust-ios-apps-ed30d673d459)
---

### 📷 Screens

<table>
  <tr>
    <th>Home Page</th>
    <th>VStack Page</th>
  </tr>
  <tr>
    <td style="padding-right: 20px;">
      <img src="https://github.com/user-attachments/assets/1eb26320-3f85-4de4-93ef-6dd533a7b659" alt="home page" width="350"/>
    </td>
    <td>
      <img src="https://github.com/user-attachments/assets/4088554c-c1a5-4108-907f-a5ac556cc2e1" alt="VStack page" width="350"/>
    </td>
  </tr>
</table>
